xmodule/video_block/transcripts_utils.py

```diff
@@ -866,11 +866,10 @@ def get_default_transcript_language(self, transcripts, dest_lang=None):
         """
         sub, other_lang = transcripts["sub"], transcripts["transcripts"]
 
-        if dest_lang:
-            if dest_lang in other_lang.keys():
-                transcript_language = dest_lang
-            elif dest_lang == 'en' and (not other_lang or (other_lang and sub)):
-                transcript_language = 'en'
+        if dest_lang and dest_lang in other_lang.keys():
```

Comment:
This is a dense block of logic; could you add a comment above it explaining what it does?